lib/delta/producer.ex

```diff
@@ -11,7 +12,7 @@ defmodule Delta.Producer do
           {:url, binary}
           | {:frequency, non_neg_integer}
           | {:http_mod, module}
-          | {:filters, [filter]}
+          | {:filters, [Filter.t()]}
```

Comment:
question: If this is moving to `Filter.t()` as the type, do you still need the `filter` type on line 16?

Reply:
fixed.